### DISCUSSION TITLE
How To Improve Performance With Multiple Web Browsers?

### DISCUSSION THREAD

**Discussion by JAng-A3** [Voltstro-Studios/UnityWebBrowser#368] [Q&A]
Hi there, I was wondering if there is any tricks or tip to optimize performance of a project that has multiple web browsers currently open? The project I am working on requires the creation of web browsers and as there are more of them the framerate of the project starts to dip exponentially.

**Comment by Voltstro** [comment]
With the currently way UWB works, no I don't think there would be any way to improve performance with multiple instances running. UWB wasn't really designed to run multiple instances.

The current way runs multiple, separate instances of CEF. The proper way would essentially be the same way tabs work, where it multiple pages but they use the same running web browser.
